SuperBuild/External_RapidJSON.cmake

```diff
@@ -21,18 +21,24 @@ if(NOT DEFINED ${proj}_DIR AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/miloyip/rapidjson.git"
+    #"${EP_GIT_PROTOCOL}://github.com/miloyip/rapidjson.git" #2023.05.09
+    "${EP_GIT_PROTOCOL}://github.com/Tencent/rapidjson.git"  #2025.06.19
```

Comment:
Updating `rapidjson` definitely makes. We will move the corresponding updates into a dedicated pull request.